aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -158,4 +190,33 @@ module aptos_std::simple_map {
 
         destroy_empty(map);
     }
+
+    struct OnlyMove has store { val: u64 }
+
+    #[test]
+    public fun upsert_test() {
```

Comment:
I wonder for inline functions, we should start separating the tests into a different module? This might help catch issues that happen on the calling sites.

Reply:
It's no longer inline